<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>1151</width>
      <height>711</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>250</y>
        <width>981</width>
        <height>210</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label1_2">
         <property name="font">
          <font>
           <pointsize>45</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Welcome to Watermarker!</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label2_2">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>To begin select an image</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Button1_2">
         <property name="font">
          <font>
           <pointsize>30</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Let's Go</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="commit_btn">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>650</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Changes</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>190</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Opacity</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>641</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>CrossCursor</cursorShape>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>650</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>80</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QDial" name="dial">
      <property name="geometry">
       <rect>
        <x>1010</x>
        <y>220</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>920</x>
        <y>-10</y>
        <width>61</width>
        <height>711</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>10</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Text</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>50</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Write your text here</string>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>180</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>100</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>140</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black</string>
      </property>
     </widget>
     <widget class="QPushButton" name="color_btn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>140</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Color</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="opacity_box">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>300</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>330</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="font_label">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>340</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Font</string>
      </property>
     </widget>
     <widget class="QPushButton" name="font_button">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>380</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Select System Font</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Custom_font_btn">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>430</y>
        <width>151</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Select Custom font</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_sys_font">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>380</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_custom_font">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>430</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QSpinBox" name="custom_font_size">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>460</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>460</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Custom Font Size</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>410</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Custom Font</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Renderbutton">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>650</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Render image</string>
      </property>
     </widget>
     <widget class="Line" name="line_5">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>490</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="font_label_2">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>500</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Watermark Settings</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="num_of_watermarks">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>540</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>540</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Watermark Num</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>570</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Rows Num</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="num_of_rows">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>570</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="watermark_rotation">
      <property name="geometry">
       <rect>
        <x>1080</x>
        <y>600</y>
        <width>41</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>360</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>970</x>
        <y>600</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Watermark Rotation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="get_result">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>650</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Get the Resulting image</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>commit_btn</zorder>
     <zorder>label_2</zorder>
     <zorder>label</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>line_2</zorder>
     <zorder>dial</zorder>
     <zorder>line</zorder>
     <zorder>label_3</zorder>
     <zorder>lineEdit</zorder>
     <zorder>line_3</zorder>
     <zorder>label_4</zorder>
     <zorder>graphicsView</zorder>
     <zorder>color_btn</zorder>
     <zorder>opacity_box</zorder>
     <zorder>line_4</zorder>
     <zorder>font_label</zorder>
     <zorder>font_button</zorder>
     <zorder>Custom_font_btn</zorder>
     <zorder>radio_sys_font</zorder>
     <zorder>radio_custom_font</zorder>
     <zorder>custom_font_size</zorder>
     <zorder>label_5</zorder>
     <zorder>Renderbutton</zorder>
     <zorder>line_5</zorder>
     <zorder>font_label_2</zorder>
     <zorder>num_of_watermarks</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>num_of_rows</zorder>
     <zorder>watermark_rotation</zorder>
     <zorder>label_8</zorder>
     <zorder>get_result</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1190</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
